packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -233,7 +241,8 @@ export default defineComponent({
       ancestorMetaData,
       sharedAncestor,
       formatDateRelative,
-      matchingSpace
+      matchingSpace,
```

Comment:
not part of your change but, could you refactor the `sharedAncestorRoute` computed into the setup function, that way you dont have to return matchingSpace since it seems to be the only reason its returned here